image/openssl/src/lib.rs

```diff
@@ -45,6 +45,12 @@ const D_LEAF: u16 = 0x8282;
 const D_INTR: u16 = 0x8383;
 
 impl ImageGeneratorCrypto for OsslCrypto {
+    fn sha256_digest(&self, data: &[u8]) -> anyhow::Result<[u32; SHA256_DIGEST_WORD_SIZE]> {
```

Comment:
This function can never fail. I don't understand why the output is wrapped in Result.

Reply:
I only did it this way because that's how sha384_digest was. I guess the idea is that in some implementations it might fail.